package.json

```diff
@@ -3,8 +3,16 @@
   "version": "3.1.62",
   "description": "jquery.inputmask is a jquery plugin which create an input mask.",
   "main": "./dist/inputmask/jquery.inputmask.js",
+  "files": [
```

Comment:
This is what files are included when doing an ``npm install jquery.inputmask``. It also bundles ``package.json``, ``README.md`` and ``CHANGELOG.md`` It would also bundle you license file, if included.
This avoid having all of you test files, scripts and minifed sources.